src/app/dynamic-forms/fields/cinchy-form-link.directive.ts

```diff
@@ -287,6 +289,12 @@ export class LinkDirective implements OnInit {
   }
 
   onInputChange() {
+    if (this.isLoading) {
+      this.myControl.setValue('');
+      this.selectedValue = null;
```

Comment:
is setting the myControl value to empty and selectedValue to null necessary? 

Also once it's done loading, it still needs the value of the control to apply the input search value without the user inputting more values. Does that work?